File: resourcespoet/src/test/java/com/commit451/resourcespoet/TypedArrayTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

import java.util.ArrayList;

/**
 * Tests the resources creation
 */
public class TypedArrayTest {

    @Test
    public void typedArrayTest() throws Exception {
        ArrayList<String> strings = new ArrayList<>();
        strings.add("One");
        strings.add("Two");

        ResourcesPoet poet = ResourcesPoet.create()
                .addTypedArray("some_typed_array", strings);

        TestUtil.assertEquals("typed_array.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/TypedArrayTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class TypedArrayTest {

    @Test
    fun typedArrayTest() {
        val strings = ArrayList<String>()
        strings.add("One")
        strings.add("Two")

        val poet = ResourcesPoet.create()
            .addTypedArray("some_typed_array", strings)

        TestUtil.assertEquals("typed_array.xml", poet)
    }
}
